lib/src/scenes/fog_exp2.dart

```diff
@@ -1,7 +1,8 @@
 part of three;
 
-class FogExp2 implements Fog {
-  Color color;
+class FogExp2 extends Fog {
   num density;
-  FogExp2(num hex, [this.density = 0.00025]) : color = new Color(hex);
+  FogExp2(num hex, [this.density = 0.00025]) {
+    color = new Color(hex);
```

Comment:
This line can be reverted. It looks like the main change was removing the local `Color color`.